### PR TITLE
ChangeLog for 3.1.0

### PR DESCRIPTION
**Pull request by WeinaJi**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,6 +2,34 @@
 Change Log
 ==========
 
+Version 3.1.0
+=============
+**12 Mar 2024**
+
+*New Features*
+  * [BBPGLIB-1102] (Part two) Load memory balance from file (#129)
```
Comment (st4rl3ss) [inline CHANGES.rst:10]:
```suggestion
  * [BBPBGLIB-1102] (Part two) Load memory balance from file (#129)
```

**Review by st4rl3ss — APPROVED** [review]
LGTM

**Review by jorblancoa — APPROVED** [review]

**Comment by atemerev** [comment]
Looks great!

**Review by atemerev — APPROVED** [review]
LGTM!

**Pull request merged**